.github/workflows/security_npm_dependency.yml

```diff
@@ -0,0 +1,12 @@
+name: Security npm dependency check
+on:
+  workflow_dispatch:
+  schedule:
+    - cron: "22 9 * * MON-FRI" # Every weekday at 09:22 UTC
```

Comment:
Could we manually edit these times so all the checks happen before 8am for all CASs? I can see why we might want them staggered (that would be a whole lot of alerts flooding Slack and other channels at once otherwise!) but it probably would be nice to be able to start the work day with all needed alerts ready to be consulted :)